scrooge-generator-typescript/src/main/scala/com/gu/scrooge/backend/typescript/ScalaObjectHandler.scala

```diff
@@ -36,7 +35,7 @@ class ScalaObjectHandler extends ReflectionObjectHandler {
     iteration: Iteration,
     writer: Writer,
     value: AnyRef,
-    scopes: Array[AnyRef]
+    scopes: List[AnyRef]
```

Comment:
does this signature change constitute a breaking change for the consumers of the library?

Reply:
I'm not sure, but this signature change was forced by the scrooge version update so I don't think there's any way around it